release.py

```diff
@@ -0,0 +1,378 @@
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+            sys.exit(1)
+    if step_complete("newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping add news fragment...{bcolors.ENDC}")
+    else:
+        try:
+            mkfile("newsfragments/{ticket}.minor".format(ticket=TICKET))
+            record_step("newsfragment_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to add newsfragment file! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+            sys.exit(1)
+    if step_complete("commit_newsfragment_complete"):
+        print(f"{bcolors.OKCYAN}Skipping commit fragment...{bcolors.ENDC}")
+    else:
+        try:
+            subprocess.run(
+                ["git", "add", "newsfragments/{ticket}.minor".format(ticket=TICKET)],
+                check=True,
+            )
+            subprocess.run(
+                ["git", "commit", "-s", "-m", "tahoe-lafs-{tag} news".format(tag=TAG)],
+                check=True,
+            )
+            record_step("commit_newsfragment_complete")
+        except Exception as e:
+            print(
+                f"{bcolors.FAIL}INFO: Failed to commit newsfragment! :(...{bcolors.ENDC}"
+            )
+            print(f"{bcolors.FAIL} {e} {bcolors.ENDC}")
+            sys.exit(1)
+    updated_content = None
+    LINE = "=" * len(RELEASE_TITLE)
+    with open("NEWS.rst", "r") as f:
+        content = f.read()
+        updated_content = re.sub(
+            r"\.\.\stowncrier start line\n(Release\s(\d+\.\d+\.\d)(\.)post\d+\s\(\d{4}-\d{02}-\d{02}\))+(\n\'+)",
+            RELEASE_TITLE + "\n" + LINE,
+            content,
+        )
+    with open("updated_news.rst", "w") as f:
+        f.write(updated_content)
+    shutil.move("updated_news.rst", "NEWS.rst")
+    print(f"{bcolors.OKGREEN}First set of release steps completed! :).{bcolors.ENDC}")
+    print(
+        f"{bcolors.OKBLUE}Instruction: Move (cd) into the  release folder : {RELEASE_FOLDER}.{bcolors.ENDC}"
+    )
+    print(
+        f"{bcolors.OKBLUE}Instruction: Please review News.rst (in release folder) {bcolors.ENDC}"
+    )
+    print(
+        f'{bcolors.OKBLUE}Instruction: Update "docs/known_issues.rst" (if necessary){bcolors.ENDC}'
+    )
+    print(
+        f'{bcolors.OKBLUE}Instruction: If any, commit changes to NEWS.rst and  "docs/known_issues.rst" {bcolors.ENDC}'
+    )
+    print(
+        f"{bcolors.OKBLUE}Instruction: If your intention IS NOT to push to \"origin (git@github.com:tahoe-lafs/tahoe-lafs.git)\", please add and set the '--repo' (example : --repo git@github.com:meejah/tahoe-lafs.git) flag to release complete command before executing"
+    )
+    print(
+        f"{bcolors.OKBLUE}Instruction: If you did not provide your signing key via the '--sign' flag (example : --sign XXXXXXXXSAMPLEYKEYXXXXXXXXX), you would need add it to command printed below"
```

Comment:
Similar to above, if `--sign` _was_ provided already we don't need the user to read this (right?)